examples/exec.js

```diff
@@ -43,14 +43,14 @@ const run = () => {
 
     strategy.ws = ws
 
-    const strategyOpts = {
+    const strategyOptions = {
       symbol: SYMBOLS.EOS_USD,
       tf: TIME_FRAMES.ONE_DAY,
       includeTrades: true,
       seedCandleCount: 5000
     }
```

Comment:
seems this properties also should be renamed

Reply:
done [23ba66e](https://github.com/bitfinexcom/bfx-hf-strategy-exec/pull/23/commits/23ba66e3edeba3d87642e835a0b289bf79683d9b)